servo/connectors/kubernetes.py

```diff
@@ -2906,7 +2906,7 @@ async def handle_error(self, error: Exception) -> bool:
                         f"cannot rollback a tuning Pod: falling back to destroy: {error}"
                     )
 
-                await asyncio.wait_for(self.destroy(), timeout=self.timeout.total_seconds())
+                await asyncio.wait_for(self.destroy(), timeout=self.timeout.total_seconds() + 60)
```

Comment:
@linkous8 Is this to avoid 0 timeout?

Reply:
Yeah what's up with the + 60 here?

Reply:
The canary pod destroy was timing out consistently even though the timeout for the test is 20 seconds. I didn't want to extend the config timeout as that will just make the test run time longer so I added a bit of extra buffer for it since its error handling logic.

Would you prefer the configured timeout be extended or should we perhaps remove the timeout from the handle_error ops?

Reply:
I don't think that removing the timeout is safe -- I don't want us waiting forever. Unless maybe there's already a higher-order timeout in effect? I've tried to unwind nested overlapping timeouts but maybe this is another such case that I missed.

I really don't like the arbitrary slop -- it seems like something is off target

Reply:
It's really taking more than 20s to destroy this Pod?

Reply:
Apparently so. You can see it in this failed test run: https://github.com/opsani/servox/runs/2677588629

```
fut = <Task cancelled name='Task-464' coro=<CanaryOptimization.destroy() done, defined at /home/runner/work/servox/servox/servo/connectors/kubernetes.py:2893>>
timeout = 20.0

...

>               raise exceptions.TimeoutError()
E               asyncio.exceptions.TimeoutError
```

Reply:
What about just boosting the timeout on the config within your test body?

Reply:
Done. (May need to boost it further if this next test still fails)